### PR TITLE
Fixed tutorials code snippets

### PR DESCRIPTION
**Pull request by Rockdeldiablo**
there was a missing plot in the logit example page and the code snippets missed the listing the  libraries needed  to run

### CODE REVIEW

**Review by pat-alt — COMMENTED** [review]
Thanks, Pasquale, but the Markdown documents are rendered through Quarto (**.qmd), so any changes should be made there. Will explain during our meeting this morning.

**Comment by Rockdeldiablo** [comment]
> Thanks, Pasquale, but the Markdown documents are rendered through Quarto (**.qmd), so any changes should be made there. Will explain during our meeting this morning.

@pat-alt  the page " a note on the priors" is incomplete and the code is hidden. Should i leave it as it is now or i can set the code to be visible?

ps. have you tried some other distribution for the priors? i have read that the normal distribution is not optimal and the student-t distribution gives better results. maybe we can test it later.

**Comment by pat-alt** [comment]
Feel free to unhide the code, if that makes it clearer.

> ps. have you tried some other distribution for the priors? i have read that the normal distribution is not optimal and the student-t distribution gives better results. maybe we can test it later.

I'm not sure how this would work, since we always assume a multivariate Gaussian posterior (see Murphy (2022) chapter 4.6.8.2). But if you have an idea, happy to discuss.

**Comment by Rockdeldiablo** [comment]
> Feel free to unhide the code, if that makes it clearer.
> 
> > ps. have you tried some other distribution for the priors? i have read that the normal distribution is not optimal and the student-t distribution gives better results. maybe we can test it later.
> 
> I'm not sure how this would work, since we always assume a multivariate Gaussian posterior (see Murphy (2022) chapter 4.6.8.2). But if you have an idea, happy to discuss.

yeah i had forgotten the laplace approx is a plugin approximation over a standard nn, rather than a fully bayesian nn.  i am not sure but Maybe the approximation of the valley in the loss function can be improved if instead a mv normal ones use the multivariate student t distribution and use backprop to finetune the degree of freedom... just an idea...

**Comment by Rockdeldiablo** [comment]
btw the command was quarto render docs/ from the laplaceredux main folder right?

**Comment by Rockdeldiablo** [comment]
sorry for the delay but there was an installation issue with vs code jupyter and quarto and the render kept failing

**Review by pat-alt — APPROVED** [review]
Thanks!

**Comment by Rockdeldiablo** [comment]
@pat-alt  can you tell me again how to generate the .qmd files from the markdown? i have made changes to regression.md but i don't remember the quarto command to update the equivalent qmd files.
